tests/ludwig/config_validation/test_validate_config_misc.py

```diff
@@ -417,3 +418,21 @@ def test_decoder_descriptions():
         type_data = feature_schema["then"]["properties"]["decoder"]["properties"]["type"]
         assert len(type_data["enumDescriptions"].keys()) > 0
         assert set(type_data["enumDescriptions"].keys()).issubset(set(type_data["enum"]))
+
+
+def test_deprecation_warning_raised_for_unknown_parameters():
+    config = {
+        "input_features": [
+            category_feature(encoder={"type": "dense", "vocab_size": 2}, reduce_input="sum"),
+            number_feature(),
+        ],
+        "output_features": [binary_feature()],
+        "combiner": {
+            "type": "tabnet",
+            "unknown_parameter_combiner": False,
+        },
+        TRAINER: {
+            "epochs": 1000,
+        },
+    }
```

Comment:
Is there any pytest util that can check for a warning raised similar to `with pytest.raises`

Reply:
yes there is! forgot to add that 👍🏽